dali/operators/image/crop/bbox_crop.cc

```diff
@@ -525,22 +525,25 @@ class RandomBBoxCropImpl : public OpImplBase<CPUBackend> {
     }
     tp.RunAll();
 
-    auto &anchor_out = ws.template OutputRef<CPUBackend>(0);
-    anchor_out.Resize(uniform_list_shape(num_samples, {ndim}));
-    auto anchor_out_view = view<float>(anchor_out);
+    DALIDataType output_dtype = has_crop_shape_ ? DALI_INT32 : DALI_FLOAT;
+    TYPE_SWITCH(output_dtype, type2id, type, (int, float), (
```

Comment:
```suggestion
    TYPE_SWITCH(output_dtype, type2id, T, (int, float), (
```
would be more in line with other parts of the codebase.

Reply:
changed